markdown.dtx

```diff
@@ -5209,6 +5209,142 @@ defaultOptions.contentBlocks = false
 %</lua,lua-cli>
 %<*manual-options>
 
+#### Option `contentType`
+
+`contentType` (default value: `block`)
+
+% \fi
+% \begin{markdown}
+%
+% \Optitem[block]{contentType}{\opt{inline}, \opt{block}}
+%
+:    inline
+
+     :  Treat all content as inline elements. Blank lines are
+        replaced with spaces.
+
+        ``` md
+        - this is a text
+        - not a list
+        ``````
+
+:    block
+
+     :  Render block and inline elements normally, 
+        no behaviour is changed.
+
```

Comment:
```suggestion
% \Optitem[block]{contentType}{\opt{block}, \opt{inline}}
%
:    block

     :  Treat content as a sequence of blocks.

        ``` md
        - this is a list
        - it contains two items
        ``````

:    inline

     :  Treat all content as inline content.

        ``` md
        - this is a text
        - not a list
        ``````

```